README.md

```diff
@@ -18,13 +18,36 @@ KubeVirt Ansible consists of a set of Ansible playbooks that deploy fully functi
 
 ## Usage
 
+### Deploy
 To deploy KubeVirt on an existing OpenShift cluster run the command below. For more information on clusters and other deployment scenarious see [playbooks instructions](./playbooks/README.md).
 
 ```
 ansible-playbook -i localhost playbooks/kubevirt.yml -e@vars/all.yml
 ```
 >**Note:** Check default variables in [vars/all.yml](./vars/all.yml) and update them if needed.
 
+### E2E Testing
+
+After a successful deployment or using an exist cluster, make sure you can `oc login` into cluster.
+
+```
+make generate-tests
+```
+Will compile tests from tests inside docker container and copy it to _out directory inside kubevirt-ansible repository.
```

Comment:
* Tests from Tests? Tests are already inside and some will be compiled in the same container? It's a very confusing phrase. Maybe: `Compile tests which are inside THE docker container`?
* s/to _out directory/to the `_out` directory
* s/kubevirt-ansible repository/the `kubevirt-ansible` repository